docs/Tutorials/Private-Network/Create-IBFT-Network.md

```diff
@@ -349,3 +349,15 @@ When finished using the private network, stop all nodes using ++ctrl+c++ in each
 
 <!-- Acronyms and Definitions -->
 *[Byzantine fault tolerant]: Ability to function correctly and reach consensus despite nodes failing or propagating incorrect information to peers.
+
+## Next steps
+
+You can continue by installing:
+
+- [Alethio Ethereum Lite Explorer](https://besu.hyperledger.org/en/stable/HowTo/Deploy/Lite-Block-Explorer/)
```

Comment:
markdown linter is complaining about this list `Unordered list style [Expected: asterisk; Actual: dash]`
if you change the dashes to * I think it will be happy :)